### PR TITLE
refactor(pillar): sync map.jinja with template-formula

### PR DESCRIPTION
**Pull request by n-rodriguez**

### CODE REVIEW

File: timezone/init.sls [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+    - name: {{ timezone.name }}
+    - utc: {{ timezone.utc }}
 
-  {%- if grains.os not in ('MacOS', 'Windows') %}
+{%- if grains.os not in ('MacOS', 'Windows') %}
 
 timezone_packages:
   pkg.installed:
-    - name: {{ confmap.pkgname }}
+    - name: {{ timezone.pkgname }}
```
Comment (noelmcloughlin) [inline timezone/init.sls]: This should be `timezone.pkg.name` because `pkg` will be a dict in template-formula.

File: timezone/init.sls [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 timezone_symlink:
   file.symlink:
-    - name: {{ confmap.path_localtime }}
-    - target: {{ confmap.path_zoneinfo }}{{ timezone }}
+    - name: {{ timezone.path_localtime }}
+    - target: {{ timezone.path_zoneinfo }}{{ timezone.name }}
     - force: true
     - require:
-      - pkg: {{ confmap.pkgname }}
+      - pkg: {{ timezone.pkgname }}
```
Comment (noelmcloughlin) [inline timezone/init.sls]: same

**Review by noelmcloughlin — CHANGES_REQUESTED** [review]
Looks pretty good @n-rodriguez  thank you for PR
I have just one requested change.  
Could you update  `pillar.example` showing `pkg.name` instead of `pkgname` too.

**Comment by n-rodriguez** [comment]
>  Could you update `pillar.example` showing `pkg.name` instead of `pkgname` too.

Well.. actually the goal of this PR was to resync with template formula without introducing any changes that could break behaviors.

**Review by noelmcloughlin — APPROVED** [review]
Thanks @n-rodriguez !!

**Pull request merged**